src/ui/dashboard/DashboardPage.js

```diff
@@ -10,6 +11,9 @@ const DashboardPage = () => (
         <Col md={4}>
           <UserManagementContainer />
         </Col>
+        <Col md={4}>
```

Comment:
use xs attribute

Reply:
why? ux is fine with those collapsing one on top of each other